scipy/linalg/tests/test_batch.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from numpy.testing import assert_allclose
+from scipy import linalg
+
+
+class TestMatrixInScalarOut:
+
+    def batch_test(self, fun, args=(), kwargs=None, dtype=np.float64,
```

Comment:
This structure will let us parametrized over `args`/`kwargs`/`dtype` for each function separately. We can add more generality as needed.